tests/test_basic.py

```diff
@@ -333,7 +333,7 @@ def test():
     client = app.test_client()
     rv = client.get('/')
     assert 'set-cookie' in rv.headers
-    match = re.search(r'\bexpires=([^;]+)(?i)', rv.headers['set-cookie'])
+    match = re.search(r'(?i)\bexpires=([^;]+)', rv.headers['set-cookie'])
```

Comment:
A couple other Pallets projects have merged patches extracting the flags. For example, pallets/werkzeug# 1053. If you're interested in creating a similar one for Flask that would be appreciated.